batch_job/src/batch_job_work_queue.c

```diff
@@ -24,10 +24,14 @@ static void specify_files(struct work_queue_task *t, const char *input_files, co
 			p = strchr(f, '=');
 			if(p) {
 				*p = 0;
-				work_queue_task_specify_file(t, f, p + 1, WORK_QUEUE_INPUT, caching_flag);
+				if(!work_queue_task_specify_file(t, f, p + 1, WORK_QUEUE_INPUT, caching_flag)){
+					fatal("Failed to specify file %s->%s in Work Queue", f, p+1);
```

Comment:
Message is confusing, as the file was specified. You want something like: Error while specifying file.